api/src/main/java/de/adorsys/keymanagement/api/metadata/KeyMetadataOper.java

```diff
@@ -8,15 +8,52 @@
 
 public interface KeyMetadataOper {
 
+    /**
+     * By default password is null for keys metadata
```

Comment:
`Password that will be used to store metadata as key entry in encrypted KeySource (i.e. java Keystore).  By default password is null for keys of metadata type`

Reply:
done